.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,79 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit-standalone:
+    name: "PHPUnit on standalone servers"
+    runs-on: "${{ matrix.os }}"
+
+    strategy:
+      matrix:
+        os:
+          - "ubuntu-20.04"
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        mongodb-version:
+          - "4.4"
+          - "4.2"
+          - "4.0"
+          - "3.6"
+        driver-version:
+          - "stable"
+        deps:
+          - "normal"
+        include:
+          - deps: "low"
+            os: "ubuntu-20.04"
+            php-version: "7.2"
+            mongodb-version: "3.6"
+            driver-version: "1.5.0"
+
+    services:
+      mongodb:
+        image: "mongo:${{ matrix.mongodb-version }}"
+        ports:
+          - "27017:27017"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+        with:
+          fetch-depth: 2
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          php-version: "${{ matrix.php-version }}"
+          tools: "pecl"
+          extensions: "mongodb-${{ matrix.driver-version }}, bcmath"
+          coverage: "none"
+          ini-values: "zend.assertions=1"
+
+      - name: "Cache dependencies installed with composer"
+        uses: "actions/cache@v2"
+        with:
+          path: "~/.composer/cache"
+          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
+          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
+
+      - name: "Install dependencies with composer"
+        run: "composer update --no-interaction --no-progress --no-suggest"
```

```suggestion
        run: "composer update --no-interaction --no-progress"
```

`--no-suggest` is deprecated: https://github.com/doctrine/mongodb-odm/pull/2250/checks#step:5:4